### DISCUSSION TITLE
Different energy range in the plot

### DISCUSSION THREAD

**Discussion by pragyan19** [SMTG-UCL/sumo#238] [Q&A]
Hi,

For my DOS calculation I gave the energy range -8 to +8. The DOSCAR file is also written accordingly. However, when plotting using sumo-dosplot, I'm giving energy range of -6 to +6 but it writes .dat files from -3.29 to +6 and shows the plot as per .dat files. The region from -6 to -3.29 remains blank. Please see the attached png file. 

How can I rectify that? 
![dos](https://github.com/SMTG-Bham/sumo/assets/128905019/ad3663d6-2970-4b06-a9b4-c3b4dda55769)
Thanks for the help

**Comment by ajjackson** [comment]
What's the Fermi energy? I think the -8 to 8 eV range for the DOSCAR is relative to some DFT reference while the -6 to +6 range for sumo is after the reference energy shift. Perhaps shifting the VBM to zero has moved the data out of range.
